### ISSUE TITLE
Selectable Assembly Packing [Feature]

### ISSUE THREAD

**Issue by MurariAlex** [mkaring/ConfuserEx#23]
The compressor doesn't allow to select which assemblies you do want to pack into your executable module.

Let's say you have 5 DLLs, but you only want the compressor to pack three of them into your EXE, the other two you want as-is, unpacked.

Today all DLLs are packed. 
This issue is to request a feature to allow you to choose which assemblies will be packed.

**Comment by mkaring** [comment]
I got this on my todo list. I actually want to extend this to a ILMerge like feature as alternative to the compressor, where both can be activated per assembly.

**Comment by mkaring** [comment]
Thinking about it some more it may also be handy if it can be declared what the target assembly for the packer is. So you don't have to pack everything into the main assembly. 
This way it would be covered if you have a applications with multiple executables that use the same “main“ assembly. For example how dnspy does it with the 64bit and the 32bit executable.

**Comment by MurariAlex** [comment]
@mkaring Agree. This would provide the necessary flexibility when packing the application, specially when dealing with multi-platform executables. I think it would be great!
